### PR TITLE
Apply copy updates

### PR DESCRIPTION
**Pull request by mtruj013**
## Done

- Updated copy as per docs linked below
- Drive by:
   - Bumped to latest Vanilla and applied `has-misaligned-images` class to broken logo sections

## QA

- Visit [landing](https://jp-ubuntu-com-476.demos.haus/) and check against [doc](https://docs.google.com/document/d/1J4zHHJ19ueONPtgj9oHTWGvWRX5JTmHy8VuGy4mygp8/edit)
- Visit /pricing and check against [doc](https://docs.google.com/document/d/1P622ZfdiwpoM0cnBfE-lSVTOMnZ44n-K5ZSgkwilM2Y/edit)
- Visit /ai-ml and check against [doc](https://docs.google.com/document/d/186nJQj0P5NTZR3CKnPkjq5I8kupCy1boHnOoHQZAHNI/edit#heading=h.g971ep4qjkem)
- Visit /k8s, /kubernetes, and /openstack and see that logo sections look as expected

## Issue / Card
https://warthogs.atlassian.net/browse/WD-10548

### CODE REVIEW

File: templates/ai-ml.html [hunk 1]
```diff
@@ -34,18 +34,30 @@ <h2>UbuntuでAIモデルを開発する</h2>
...
-        <div class="p-logo-section__item"><img class="p-logo-section__logo" src="https://res.cloudinary.com/canonical/image/fetch/q_auto,f_auto/https://assets.ubuntu.com/v1/d3ee88a5-2018-logo-tesla.svg" alt="Tesla"></div>
-        <div class="p-logo-section__item"><img class="p-logo-section__logo" src="https://res.cloudinary.com/canonical/image/fetch/q_auto,f_auto/https://assets.ubuntu.com/v1/c346c103-openai.png" alt="OpenAI"></div>
-        <div class="p-logo-section__item"><img class="p-logo-section__logo" src="https://res.cloudinary.com/canonical/image/fetch/q_auto,f_auto/https://assets.ubuntu.com/v1/9eeeebca-logo-ibm.svg" alt="IBM"></div>
-        <div class="p-logo-section__item"><img class="p-logo-section__logo" src="https://res.cloudinary.com/canonical/image/fetch/q_auto,f_auto/https://assets.ubuntu.com/v1/79974738-facebook-1.svg" alt="Facebook"></div>
+        <div class="p-logo-section__item"><img class="p-logo-section__logo" src="https://assets.ubuntu.com/v1/0adc7049-jupyter-logo.png" alt="Jupyter"></div>
```
Comment (akbarkz) [inline templates/ai-ml.html:42]: Can you not make it look similar to how it looks at https://ubuntu.com/ai? Right now the logos are too small.

Comment (mtruj013) [reply templates/ai-ml.html:42]: Bumping to latest vanilla seems to have done the trick. Fixed a few other broken logo sections as well

**Review by carkod — APPROVED** [review]
So much stuff doesn't much the copy docs, but I guess updating what is requested is what is being done here, so LGTM.

**Comment by mtruj013** [comment]
> By the way, /k8s is 404, but it doesn't seem to be linked anywhere on this site.

Typo, I changed it to `/kubernetes` after I remembered that this site doesn't have that redirect in place, but forgot to delete the original 😅 . Anyways, thanks for the review!

**Pull request merged**